src/layout/Accordion/Accordion.tsx

```diff
@@ -4,21 +4,26 @@ import { Accordion as DesignSystemAccordion } from '@digdir/design-system-react'
 import { Grid } from '@material-ui/core';
 
 import { useLanguage } from 'src/hooks/useLanguage';
+import classes from 'src/layout/Accordion/Accordion.module.css';
 import { AccordionItem } from 'src/layout/Accordion/AccordionItem';
 import { GenericComponent } from 'src/layout/GenericComponent';
 import type { PropsFromGenericComponent } from 'src/layout';
 
-type IAccordionProps = PropsFromGenericComponent<'Accordion'>;
+type IAccordionProps = PropsFromGenericComponent<'Accordion'> & { renderPDFPreview?: boolean };
 
-export const Accordion = ({ node }: IAccordionProps) => {
-  const { textResourceBindings, renderAsAccordionItem } = node.item;
+export const Accordion = ({ node, renderPDFPreview = true }: IAccordionProps) => {
```

Comment:
The warning here says `renderPDFPreview` is not used anywhere - was this accidentally committed?

Reply:
Ah yes. This was an early idea. Will remove the unused prop.